modules/utility/init.zsh

```diff
@@ -175,9 +175,13 @@ elif (( $+commands[wget] )); then
   alias get='wget --continue --progress=bar --timestamping'
 fi
 
-# Resource Usage
-if (( $+commands[pydf] )); then
-  alias df=pydf
+if zstyle -s ':prezto:module:utility' pydf 'yes'; then
+  # Resource Usage
+  if (( $+commands[pydf] )); then
+    alias df=pydf
+  else
+    alias df='df -kh'
+  fi
```

Comment:
Possible to combine to two `if` conditions into a _single_ one? It might avoid unnecessary nested `if` and duplicate `alias df='df -kh'` in multple `else` blocks.